### PR TITLE
Minor changes to better support custom issue trackers.

### PR DESCRIPTION
**Pull request by oliverchang**
- Pass config to issue tracker constructor.
- Add a clear() method to LabelStore.

### CODE REVIEW

File: src/appengine/libs/issue_management/issue_tracker.py [hunk 1]
```diff
@@ -74,6 +74,11 @@ def reset(self):
     self._added.clear()
     self._removed.clear()
 
+  def clear(self):
+    """Clear the store, and reset tracking."""
+    self.reset()
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/issue_tracker.py]: Do we need both reset and clear function. Can we clear backing in all cases. Just the two function names are similar and confusing and i wonder if people will incorrectly use one vs other.

Comment (oliverchang) [reply src/appengine/libs/issue_management/issue_tracker.py]: Renamed reset to reset_tracking to make the difference clearer. We do need both.

**Pull request merged**